src/sentry/runner/commands/cleanup.py

```diff
@@ -310,6 +319,42 @@ def is_filtered(model):
 
                 task_queue.join()
 
```

Comment:
I suggest we reduce the `DELETES_BY_PROJECT` before jumping into the project for loop to avoid `is_filtered` in each iteration or avoiding the whole part if cleaning up other models. Something like:

```python
to_delete_by_project = []
for model in DELETES_BY_PROJECT:
      if is_filtered(model):
          if not silent:
              click.echo(">> Skipping %s" % model.__name__)
      else:
          to_delete_by_project.append(model)

if to_delete_by_project:
    ... do the deletions for each project ....
else: # not needed
   .... skip the whole part ....
```